### DISCUSSION TITLE
Is it correct to pick the most abundance cell type?

### DISCUSSION THREAD

**Discussion by Jaimomar99** [BayraktarLab/cell2location#254] [Q&A]
First of all, thank you for creating such amazing tool, really helpful.

My goal is to run Cellchat in the spatial data, so I ran cell2location to know which cell types are in the spots. But,  is it correct to take the most abundant cell type?? Supposing we have the most abundant with 0.45 and the second with 0.43, taking the first one will trigger to a wrong analysis. 

I have come up with look at the most expressed genes in those spots that are not clear and then decide the cell type, but I am open for more ideas
Thanks in advance

**Comment by vitkl** [comment]
The short answer is No. Each location really contains multiple cell types. It is incorrect to treat locations as cells and as cell types. Consider this approach - https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html#Estimate-cell-type-specific-expression-of-every-gene-in-the-spatial-data-(needed-for-NCEM)
